index.html

```diff
@@ -9,6 +9,29 @@
         <link rel="stylesheet" href="main.css">
     </head>
     <body>
-        
+        <nav>
+            <p>Welcome</p>
+            <a href="#"><img src="img/Icon-Menu.svg" alt="Menu"></a>
+        </nav>
+        <section class="headline">
+            <header>
+                <h2>Hey there. I'm Saadat 
+                    <br>
+                    <span>I'm a software developer</span>
+                </h2> 
+            </header>
```

Comment:
- [ ] I suggest you replace this `header` tag with a `div`. This is because it is better to have the `header` tag to wrap your logo and menu. See my suggestion below:

```suggestion
            <div>
                <h2>Hey there. I'm Saadat 
                    <br>
                    <span>I'm a software developer</span>
                </h2> 
            </div>
```